Formula/got.rb

```diff
@@ -0,0 +1,35 @@
+class Got < Formula
+  desc "Version control system"
+  homepage "https://gameoftrees.org"
+  url "https://gameoftrees.org/releases/portable/got-portable-0.75.tar.gz"
+  sha256 "b35324893c9aefbd6bf49602a6c9bb9d6e20670aff3c9734fefead0205bb9cb2"
+  license "ISC"
+
+  depends_on "bison" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libevent"
+  depends_on :macos # FIXME: ossp-uuid and util-linux conflict with each other.
```

Comment:
Ok, I don't know how to get this to work on Linux. We can ship this on macOS for now.